README.md

```
@@ -264,7 +264,10 @@ make build
 ### Default Signal Chain
 The application starts with a clean acoustic preset. Only EQ and Reverb are enabled by default — all other effects start bypassed:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)